challenge_set_a_table/src/challenge_set_a_table_states/manipulate_machine.py

```diff
@@ -178,9 +179,11 @@ def __init__(self, robot, grasp_designator1, grasp_designator2, grasp_designator
         grasp_furniture_designator2 = ds.EntityByIdDesignator(robot, id=grasp_furniture_id2)
 
         place_furniture_designator = ds.EntityByIdDesignator(robot, id=place_furniture_id)
-        place_designator = ds.EmptySpotDesignator(robot=robot,
-                                                  place_location_designator=place_furniture_designator,
-                                                  area="on_top_of")
+        arm_designator = ds.OccupiedArmDesignator(robot, {})
```

Comment:
Why an OccupiedArmDesignator? All the others have an UnOccupiedArmDesignator

Reply:
Depends on the situation. In this case however there is no reason to choose either an occupied or unoccupied designator. So I changed it to a general armdesignator. This could be integrated better in the challenge if we know which arm we will use for the operation but that would require API breaks in the challenge. I think we should only look at that if we start working on this challenge.

Reply:
Fair enough IMO.